.github/workflows/tauri_ci.yaml

```diff
@@ -32,6 +32,37 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
 
+      - name: Cache Rust dependencies
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: rust-${{ runner.os }}
+
+      - name: Cache rust-lib dependencies
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: rust-lib-${{ runner.os }}-${{ github.ref }}
+          workspaces: frontend/rust-lib
+
+      - name: Cache src-tauri dependencies
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: frontend/appflowy_tauri/src-tauri
+          key: tauri-${{ runner.os }}-${{ github.ref }}
+
+      - name: Cache Node.js dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-${{ runner.os }}
+
+      - name: Cache node_modules
+        uses: actions/cache@v2
+        with:
+          path: frontend/appflowy_tauri/node_modules
+          key: node-modules-${{ runner.os }}-${{ github.ref }}
+          restore-keys: |
+            node-modules-${{ runner.os }}-
```

Comment:
Add a comment to describe why we should separate them instead of combining all the paths into the 'workspaces'.